src/common/balances.lua

```diff
@@ -8,15 +8,15 @@ function balances.walletHasSufficientBalance(wallet)
 end
 
 function balances.transfer(to)
-	utils.validateArweaveId(to)
+	assert(type(to) == "string", "'To' must be a string")
```

Comment:
this seems like another change related to ETH address support that doesn't pertain to registry callback logic we've discussed. please scope this PR accordingly.